sled-agent/src/bootstrap/pre_server.rs

```diff
@@ -521,21 +543,22 @@ impl BootstrapNetworking {
         })
     }
 
-    async fn enable_ipv6_forwarding() -> Result<(), StartError> {
-        tokio::task::spawn_blocking(|| {
```

Comment:
AFAICT this was spawning a new task to run a synchronous command? I'm simply using the async variant of `execute` here instead.